test/hpax_test.exs

```diff
@@ -86,6 +136,10 @@ defmodule HPAXTest do
     end
   end
 
+  defp header_with_store() do
+    bind(header(), fn {name, value} -> {:store, constant(name), constant(value)} end)
```

Comment:
You can do

```suggestion
    map(header(), fn {name, value} -> {:store, name, value} end)
```

Reply:
Thanks - I've not used StreamData before so I was mostly just copying from existing work in the file. Will update